bin/seeds.js

```diff
@@ -1 +1,48 @@
-// Iteration #1
+//require mongoose so we can use mongoosey things
+const mongoose = require('mongoose');
+//require our drone schema
+const Drone = require('../models/Drone.model');
+//variable containing our db name
+const DB_NAME = 'droneDB';
+
+//connecting to the db
+mongoose.connect(`mongodb://localhost/${DB_NAME}`, {
```

Comment:
Same as previous exercise, it's a good practice if you start placing this connection string in the .env file